### ISSUE TITLE
Add label to slack form

### ISSUE THREAD

**Issue by matthew-gerstman** [usereactnyc/usereact.nyc#36]
Slack form does not have a label for a11y

**Comment by dance2die** [comment]
I ran an [Axe](https://www.deque.com/axe/) test and adding `aria-label="enter email"` passes the test.
Would you let me know if this is a way to fix the issue? 

_I will do a PR for this (not sure if this is OK as this is my first time dealing with a11y 😀)._

![fix demo](https://user-images.githubusercontent.com/8465237/60141304-de24b100-9782-11e9-8abc-e2b6f82fb11d.gif)

**Comment by matthew-gerstman** [comment]
That would be perfect, thank you!

**Comment by dance2die** [comment]
I fixed the a11y on local machine but

![image](https://user-images.githubusercontent.com/8465237/60142082-45903000-9786-11e9-9027-1cb0d58e4e00.png)

Test script fails unable to find `data/author.json` & `gatsby-node` files, which I can't seem to find in the master branch.

![image](https://user-images.githubusercontent.com/8465237/60142136-85571780-9786-11e9-9b17-ac8af5befd90.png)

Not sure if I messed up 😅

**Comment by matthew-gerstman** [comment]
Yea those tests came with Gatsby and never worked. I put up #45 to delete them
